src/command.rs

```diff
@@ -560,6 +568,33 @@ impl App {
 
                 crate::bounded::set::check(&mut m, &universe, depth, compress_traces.into());
             }
+            Command::SatCheck { depth, .. } => {
+                let mut universe = HashMap::new();
+                let stdin = std::io::stdin();
+                for sort in &m.signature.sorts {
+                    println!("how large should {} be?", sort);
+                    let input = stdin
+                        .lock()
+                        .lines()
+                        .next()
+                        .expect("no next line")
+                        .expect("could not read next line")
+                        .parse::<usize>();
+                    match input {
+                        Ok(input) => {
+                            universe.insert(sort.clone(), input);
+                        }
+                        Err(_) => {
+                            eprintln!("could not parse input as a number");
+                            process::exit(1);
+                        }
+                    }
+                }
```

Comment:
Can this reuse the command-line interface for `SetCheck`?

Reply:
It can't copy it directly, because SetCheck has other options that don't make sense here (trace compression and optional depth). I'll open an issue to factor out what can be combined.